fire/SFMainMod.F90

```diff
@@ -740,10 +741,11 @@ subroutine rate_of_spread ( currentSite, ROS_torch, passive_crown_FI, heat_per_a
           heat_per_area = ir * time_r          
 
           ! calculate torching index based on wind speed and crown fuels 
-          ! ROS for crown torch initation (m/min), Eq 18 Scott & Reinhardt 2001 
-          ROS_torch = (1.0 / 54.683 * wind_reduce)* &
-                      ((((60.0*passive_crown_FI*currentPatch%fuel_bulkd*eps*q_ig)/heat_per_area*ir*xi)-1.0) &
-                       / (c*beta_ratio)**-e)**1/b
+          ! ROS for crown torch initation (m/min), Eq 18 Scott & Reinhardt 2001
+          ! TODO slevis: Are we missing a -phi_s after the -1?
+          ROS_torch = 1._r8 / (54.683_r8 * wind_reduce) * &
+    ((60._r8 * passive_crown_FI * currentPatch%fuel_bulkd * eps * q_ig / &
+      (heat_per_area * ir * xi) - 1._r8) / (c * beta_ratio**-e))**(1._r8 / b)
```

Comment:
@jkshuman 
commenting out lines 408 and 495 got us past the divide-by-zero in eq 18; however, are we missing a `-phi_s` term after the `-1` or did you omit it intentionally?

Reply:
that is for the slope. I meant to add it in as a placeholder with a value of zero as we do not have slope currently. So you can add a parameter "slope_factor" and set it to zero.